### ISSUE TITLE
wiremock 3.x and webhook  error

### ISSUE THREAD

**Issue by defnngj** [wiremock/wiremock#2341]
### Proposal

* run error :

```
Exception in thread "main" java.lang.reflect.InvocationTargetException
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
at com.github.tomakehurst.wiremock.extension.Extensions.load(Extensions.java:180)
at java.base/java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
at java.base/java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source)
at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
at java.base/java.util.stream.StreamSpliterators$WrappingSpliterator.forEachRemaining(Unknown Source)
at java.base/java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Unknown Source)
at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source)
at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source)
at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
at java.base/java.util.stream.ReferencePipeline.forEach(Unknown Source)
at com.github.tomakehurst.wiremock.extension.Extensions.load(Extensions.java:70)
at com.github.tomakehurst.wiremock.core.WireMockApp.(WireMockApp.java:92)
at com.github.tomakehurst.wiremock.WireMockServer.(WireMockServer.java:70)
at com.github.tomakehurst.wiremock.standalone.WireMockServerRunner.run(WireMockServerRunner.java:69)
at wiremock.Run.main(Run.java:23)
Caused by: java.lang.NoSuchMethodError: 'void com.github.tomakehurst.wiremock.extension.responsetemplating.TemplateEngine.(java.util.Map, java.lang.Long, java.util.Set)'
at org.wiremock.webhooks.Webhooks.(Webhooks.java:62)
at org.wiremock.webhooks.Webhooks.(Webhooks.java:67)
```

### Reproduction steps

* dockerfile 

```
FROM wiremock/wiremock:latest

ENV WEBHOOKS_VERSION 3.0.0

COPY stubs /home/wiremock

ADD https://repo1.maven.org/maven2/org/wiremock/wiremock-webhooks-extension/$WEBHOOKS_VERSION/wiremock-webhooks-extension-$WEBHOOKS_VERSION.jar /var/wiremock/extensions/

CMD ["--global-response-templating", "--print-all-network-traffic", "--extensions", "org.wiremock.webhooks.Webhooks"]
```

latest == 3.0.0  ？ 

https://github.com/wiremock/wiremock-docker

**Comment by defnngj** [comment]
close!

**Comment by oleg-nenashev** [comment]
Also raised https://github.com/wiremock/wiremock/issues/2342 for the 3.0.0 artifact

**Comment by defnngj** [comment]
ok, back to version `2.35` for the time being, running normally.

**Comment by oleg-nenashev** [comment]
We plan to release 3.0.1 today that will at least fix the issue blocking
use of the 3.x webhook extension

**Comment by tomakehurst** [comment]
Fixed in 3.0.1
